### ISSUE TITLE
Everything is terrible and broken with pika 0.10.0

### ISSUE THREAD

**Issue by atmalagon** [project8/dripline#131]
seriously WTF?

so we should specify 0.9.14 as the dep, or upgrade our pika usage to comply with 0.10.0 (unclear if 9 and 10 can be supported simultaneously)

**Comment by laroque** [comment]
Yeah, I'm not going to mess with this right now. This seems like a good project for someone to start learning about the pika API, since it involves seeing how 0.9 and 0.10 differ and where our code breaks as a result (I'd probably just install it and try and run everything and see what happens).

**If reasonable, please try to make changes such that the code is agnostic to the pika version (that is, don't go refactoring things or putting in conditions that test on the pika version and deals with each, but it seems reasonable to keyword arguments for example)**
